R/select-tbl.R

```diff
@@ -49,6 +49,8 @@ tidyselect_table_names <- function(dm) {
 }
 
 cdm_select_tbl_impl <- function(dm, selected) {
+
+  if (is_empty(selected)) return(empty_dm())
```

Comment:
Do we really need the shortcut here?

Reply:
We get an error otherwise:
```r
Error in dot_call(capture_dots, frame_env = frame_env, named = named,  : 
  attempt to set an attribute on NULL
```

Reply:
Where does that error occur? Also with `selected = character()`?

Reply:
in `filter_recode_table(selected)` when `selected` is `named character(0)`